src/services/json-schema.draft-07.ts

```diff
@@ -0,0 +1,173 @@
+/* eslint-disable sonarjs/no-duplicate-string */
+export default {
+  $schema: 'http://json-schema.org/draft-07/schema#',
```

Comment:
Just for reference, I suggested this https://github.com/asyncapi/spec-json-schemas/issues/120 for the future.